src/screens/news/functions.js

```diff
@@ -2,11 +2,21 @@ import { useEffect, useState } from 'react';
 import { useXhr } from '../../utils/xhr/hook';
 import requests from '../../utils/xhr/requests';
 
-export function useFetchNews() {
-  const [send] = useXhr({ ...requests.news.index, showErrorSnackbar: true });
+export function useFetchNews(onlyMostRecentNews = false) {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [send] = useXhr({
+    ...requests.news.index,
+    showErrorSnackbar: true,
+    queryParams: {
+      ...requests.news.index.queryParams,
+      ...Object.assign({}, onlyMostRecentNews ? { pagination: 5 } : {}),
```

Comment:
This is redundant, the spread operator is syntax sugar for Object.assign, you can just write
```js
...(onlyMostRcentNews? { pagination: 5} : {})
```

Reply:
Thanks, I will fix it